src/executor/query/DataCollectExecutor.cpp

```diff
@@ -61,12 +61,15 @@ Status DataCollectExecutor::collectSubgraph(const std::vector<std::string>& vars
     DataSet ds;
     ds.colNames = std::move(colNames_);
     // the subgraph not need duplicate vertices or edges, so dedup here directly
-    std::unordered_set<Value> uniqueVids;
+    std::unordered_set<std::string> uniqueVids;
```

Comment:
Why change this?

Reply:
According to the benchmark of yee， The efficiency of hash(string) is higher than that of hash(value)

Reply:
Yes, but the Value(int).toString() is not efficient, I think you could use the int binary directly.

Reply:
Do you mean  Value.toInt()

Reply:
Yes.

Reply:
Value.toInt() will get wrong result when Value is string 
eg: Value("123 abc").toInt == Value("123 cde").toInt

Reply:
toString when value is string